client/components/filters/advanced/utils.js

```diff
@@ -0,0 +1,115 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { find, compact } from 'lodash';
+
+/**
+ * Get the url query key from the filter key and rule.
+ *
+ * @param {string} key - filter key.
+ * @param {string} rule - filter rule.
+ * @return {string} - url query key.
+ */
+export const getUrlKey = ( key, rule ) => {
+	if ( rule ) {
+		return `${ key }_${ rule }`;
+	}
+	return key;
+};
+
+/**
+ * Convert url values to array of objects for <Search /> component
+ *
+ * @param {string} str - url query parameter value
+ * @return {array} - array of Search values
+ */
+export const getSearchFilterValue = str => {
+	return str.length ? str.split( ',' ) : [];
+};
+
+/**
```

Comment:
Appears to be an orphaned doc block?

Reply:
Naw, its a typedef, http://usejsdoc.org/tags-typedef.html, for `activeFilter` since its a common shape I use throughout the file